README.md

```diff
@@ -1 +1,39 @@
-# Awesome-book
\ No newline at end of file
+# Awesome-book
```

Comment:
- [ ] In your README, kindly add more specific and detailed instructions on how users can set up and run your app locally.

Reply:
yes sure, we fix this 👍